specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.3/examples/AnalyzeOperationResult.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "endpoint": "{endpoint}",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "modelId": "{modelId}",
+    "resultId": "{result Id}",
+    "body": {}
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "status": "running",
+        "createdDateTime": "2019-05-01T10:53:21Z",
+        "lastUpdatedDateTime": "2019-05-01T10:53:23Z",
+        "analyzeResult": {
+          "version": "v2.1",
```

Comment:
I believe the current service returns "2.1.0" instead of "v2.1" (at least prebuilt verticals).  Can we check if we do this consistently across all our analysis operations?

Reply:
That is correct.. this is swagger validation tooling limitation i.e. adding 2.1.0 has caused some tool in verification chain to barf.